docs/Contributing/Testing-and-local-development.md

```diff
@@ -492,20 +492,43 @@ Once you have access:
 1. Click on the "Download Token" button at the top and confirm the download in the modal.
 1. Save the token in a safe place.
 
-### APNs setup
+### APNs and SCEP setup
 
-The server also needs a private key + certificate to identify with Apple's [APNs](https://github.com/fleetdm/fleet/blob/main/tools/mdm/apple/glossary-and-protocols.md#apns-apple-push-notification-service) servers.
+The server also needs a private key + certificate to identify with Apple's [APNs](https://github.com/fleetdm/fleet/blob/main/tools/mdm/apple/glossary-and-protocols.md#apns-apple-push-notification-service) servers, and another for [SCEP](https://github.com/fleetdm/fleet/blob/main/tools/mdm/apple/glossary-and-protocols.md#scep-simple-certificate-enrollment-protocol).
 
-Users will be able to generate this using `fleetctl generate mdm-apple` in the near future, but for now, ask in [#g-mdm](https://fleetdm.slack.com/archives/C03C41L5YEL) how to get them.
+Both can be generated using this command (note that the email must be a company email address, it cannot be an email from commercial email providers like `gmail.com`, so you can use your `@fleetdm.com` email):
```

Comment:
Both can be generated using the command below. The email must be a company email address. It cannot be an email address from commercial email providers like `gmail.com`.

Reply:
:+1: updated.